.github/workflows/go-unit-tests.yaml

```diff
@@ -18,6 +18,15 @@ jobs:
       matrix:
         go-version: ["1.18"]
         platform: ["ubuntu-latest"]
+        tests: ["NoRuler","Ruler","Cover"] # run those as separate job to avoid killing worker #TODO <issue>
+        include:
+          - tests: "NoRuler"
```

Comment:
I renamed the Ruler test funds, checked these flags, and they seem to work as expected. Now we find out if GitHub can run them.  I considered pulling this fancy matrix into my rename-funcs PR but decided against it because if there's some stupid little syntax problem or other bug you'd be better equipped to fix it.